.github/workflows/pr_gate.yml

```diff
@@ -61,9 +61,13 @@ jobs:
         run: |
           sudo apt-get install ninja-build
 
+      - name: Git config for fetching pull requests
+        run: |
+          git config --global --add remote.origin.fetch +refs/pull/*/merge:refs/remotes/pull/*
+
       - name: CMake config
         working-directory: ${{github.workspace}}/examples/c
-        run: cmake -B ${{github.workspace}}/examples/c/build --preset release-clang
```

Comment:
This is a very interesting solution. Could we possibly solve it in a (very slightly) different way? If we set the values for `REGOCPP_C_WRAPPER_REPO` and `REGOCPP_C_WRAPPER_TAG` via a workflow `env:` call instead of as command line arguments to CMake then I can re-use them in the Rust workflow to solve the exact same problem there.

Reply:
Refactored like this.  Looks fine like this.  I did the other way as it is more standard CMake style (IMO), but this works easier than threading parameters through build.rs for the Rust crate.